src/internal/connector/graph/cache_container.go

```diff
@@ -186,5 +186,10 @@ func CheckIDAndName(c Container) error {
 		return clues.New("container missing display name").With("container_id", id)
 	}
 
+	pfid := ptr.Val(c.GetParentFolderId())
```

Comment:
mmm may want to double check this. iirc the folder cache calls some functions like this but due to the difference between calendars and mail/contacts it can only check ID and display name since it doesn't have a concept of parent folders. I think that's why these functions were split to begin with